src/latest-release.ts

```diff
@@ -1,7 +1,6 @@
 import * as nodegit from 'nodegit';
 import * as semver from 'semver';
 
-
 export async function checkoutLatestRelease(
```

Comment:
Fair question.  This file only changed because clang-format removed an extra line, I think.  I haven't integrated/tested this feature yet.  Will address migrating this into the `git.ts` file in #6 which is the issue (still open) for adding latest-release functionality.